test/fixtures/algorithms/algorithms.js

```diff
@@ -0,0 +1,83 @@
+import path from 'path';
+import {
+  AUTHOR_USER,
+  PROGRAMMING_LANGUAGES,
+} from '../../../src/config/constants';
+
+const DEFAULT_AUTHOR = 'Graasp';
+
+// eslint-disable-next-line import/prefer-default-export
+export const GRAASP_ALGORITHMS = [
```

Comment:
I know you cannot access the constants because of `getPath`. But let's try to split the file to be able to get these constants.